tests/test_exceptiongroup.py

```diff
@@ -0,0 +1,241 @@
+from sentry_sdk.utils import event_from_exception
+
+
+try:
+    # Python 3.11
+    from builtins import ExceptionGroup  # type: ignore
+except ImportError:
+    # Python 3.10 and below
+    ExceptionGroup = None
+
+
+def test_exceptiongroup():
+    exception_groups_available = ExceptionGroup is not None
+    if not exception_groups_available:
+        return
+
+    exception_group = None
+
+    try:
+        try:
+            raise RuntimeError("something")
+        except RuntimeError:
+            raise ExceptionGroup(
+                "nested",
+                [
+                    ValueError(654),
+                    ExceptionGroup(
+                        "imports",
+                        [
+                            ImportError("no_such_module"),
+                            ModuleNotFoundError("another_module"),
+                        ],
+                    ),
+                    TypeError("int"),
+                ],
+            )
+    except ExceptionGroup as e:
+        exception_group = e
+
+    (event, _) = event_from_exception(
+        exception_group,
+        client_options={
+            "include_local_variables": True,
+        },
+        mechanism={"type": "test_suite", "handled": False},
+    )
+
+    values = event["exception"]["values"]
+
+    # For this test the stacktrace and the module is not important
+    for x in values:
+        if "stacktrace" in x:
```

Comment:
this is fine to test just the exception group nesting structure, but I would add another test with a simpler exception group to make sure the stacktrace/traceback etc is also correct

Reply:
done.